.circleci/config.yml

```diff
@@ -426,6 +426,32 @@ jobs:
             conda activate python${PYTHON_VERSION}
             python -c "import torchaudio"
 
+  smoke_test_windows_conda_gpu:
+    <<: *binary_common
+    executor:
+      name: windows-gpu
```

Comment:
nit: another way would be to parameterize the executor name, but I guess it's okay to duplicate the whole definition.